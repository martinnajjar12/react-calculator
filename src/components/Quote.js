import { makeStyles, Typography } from '@material-ui/core';
import React from 'react';

const useStyles = makeStyles(theme => ({
  quoteMargin: {
    marginTop: '10vh',
    fontSize: '2.5rem',
    [theme.breakpoints.down('xs')]: {
      fontSize: '1.5rem',
    },
  },
}));

const Quote = () => {
  const classes = useStyles();
  return (
    <Typography className={classes.quoteMargin}>
      Mathematics is not about numbers, equations, computations, or algorithms: it is
      about understanding. â€“William Paul Thurston
    </Typography>
  );
};

export default Quote;
