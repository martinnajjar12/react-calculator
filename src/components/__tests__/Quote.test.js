import renderer from 'react-test-renderer';
import Quote from '../Quote';

describe('Header component', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Quote />);
    expect(tree).toMatchSnapshot();
  });
});
