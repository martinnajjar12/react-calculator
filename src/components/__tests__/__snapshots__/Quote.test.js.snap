// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match the snapshot 1`] = `
<p
  className="MuiTypography-root makeStyles-quoteMargin-1 MuiTypography-body1"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</p>
`;
